### DISCUSSION TITLE
Month time key

### DISCUSSION THREAD

**Discussion by Domainedeletoile** [Stellarium/stellarium#232] [Q&A] [answered]
Hi,

Is it possible to have shorts keys to manage time by month ? It will be easier to show how the sky changes wihin the year.

Thnak you.

Alexandre PIGUET

**Comment by gzotti** [comment, marked as answer]
Sure. Just call the Help panel (F1) and assign your own hotkeys to the time switches.

**Comment by Domainedeletoile** [comment]
Yes, I didn't think about it.

Thank you !

Alexandre

Merci, Cordialement,

*Alexandre PIGUET*

Porteur du Projet du Domaine de l’Etoile

http://domainedeletoile.fr, Inscrivez-vous sur mon *Blog* pour suivre
l’aventure…

Et suivez moi sur *FaceBook* à https://www.facebook.com/domainedeletoile/

Tel : 06-60-13-62-89

alexandre.piguet@domainedeletoile.fr



2018-07-27 9:29 GMT+02:00 Georg Zotti <notifications@github.com>:

**Comment by alex-w** [comment]
I've added support to add/subtract calendric months also

**Comment by Domainedeletoile** [comment]
Super, where can I find this support ?

Le ven. 27 juil. 2018 à 10:56, Alexander Wolf <notifications@github.com> a
écrit :

**Comment by alex-w** [comment]
In next version for calendric month...

**Comment by Domainedeletoile** [comment]
Thank you very much ☺ ! It is what I wanted.

Merci, Cordialement,

*Alexandre PIGUET*

Porteur du Projet du Domaine de l’Etoile

http://domainedeletoile.fr, Inscrivez-vous sur mon *Blog* pour suivre
l’aventure…

Et suivez moi sur *FaceBook* à https://www.facebook.com/domainedeletoile/

Tel : 06-60-13-62-89

alexandre.piguet@domainedeletoile.fr



2018-07-27 12:33 GMT+02:00 Alexander Wolf <notifications@github.com>:
